composer.json

```diff
@@ -11,24 +11,24 @@
         }
     ],
     "config": {
-        "platform": {"php": "5.6"}
+        "platform": {"php": "7.4"}
```

Comment:
Confused: If we are bumping it why not use a version that is not EOL? Are we breaking anything by bumping this?

Reply:
Good call, I had my experiment:

- We cannot remove it as we will fail phpunit install for php version 8.0 or later. We requires phpunit to be `^5.6.3`. Error message:
`phpunit/phpunit[5.6.3, ..., 5.7.27] require php ^5.6 || ^7.0 -> your php version (8.0.28) does not satisfy that requirement.`
- We don't support latest phpunit that supports later version of php, as we are using old API, and they BREAKS API, which reminds us how bad it's for customers if you BREAK your API :)
- I bump it up as I had issue when I tried to install some other package that I was playing around with php and found out it's TOO OLD, so, I just picked one that's not TOO OLD, and seems widely used.

For the config, it doesn't really do anything, it just let composer assume you are on that version regardless of the real runtime version of the php, when composer builds the dependence for you. https://getcomposer.org/doc/06-config.md#platform

So, in all, I think we probably should update our phpunit someday to support all those versions of PHP that we claim to support. But, I don't think it's a high priority for now, we have tests working on certain runtime, and it doesn't affect our customers. I can leave it as a TODO